modules/core/src/main/scala/org/scalasteward/core/update/UpdateAlg.scala

```diff
@@ -42,7 +44,8 @@ final class UpdateAlg[F[_]](
       maybeUpdate0 = maybeNewerVersions.map { newerVersions =>
         Update.Single(CrossDependency(dependency.value), newerVersions.map(_.value))
       }
-      maybeUpdate1 = maybeUpdate0.orElse(findUpdateWithNewerGroupId(dependency.value))
+      migratedUpdate = groupMigrations.findUpdateWithNewerGroupId(dependency.value)
+      maybeUpdate1 <- OptionT.fromOption[F](maybeUpdate0).orElseF(migratedUpdate).value
```

Comment:
This will read and parse the `groupMigrations` file for every dependency that is up to date. I want `findUpdate` as fast as possible (since it is called for every dependency of each repo) and would prefer if the `groupMigrations` file could be loaded when `GroupMigrations` is created so that we're loading that file only once at start-up.

Reply:
Sure, let's do this. I think I based this on the scalafix integration, so it probably loads every time as well

Reply:
You're right. I'll change the Scalafix code so that the migrations are only loaded once, too.